cmd/blockchain/ethereum/diaOracleV2MultiupdateService/main.go

```diff
@@ -154,17 +155,28 @@ func main() {
 		log.Fatalf("Failed to connect to the Ethereum client: %v", err)
 	}
 
+	connBackup, err := ethclient.Dial(backupNode)
+	if err != nil {
+		log.Fatalf("Failed to connect to the backup Ethereum client: %v", err)
+	}
+
 	auth, err := bind.NewTransactorWithChainID(strings.NewReader(key), key_password, big.NewInt(chainId))
 	if err != nil {
 		log.Fatalf("Failed to create authorized transactor: %v", err)
 	}
 
-	var contract *diaOracleV2MultiupdateService.DiaOracleV2MultiupdateService
+	var contract, contractBackup *diaOracleV2MultiupdateService.DiaOracleV2MultiupdateService
+
 	err = deployOrBindContract(deployedContract, conn, auth, &contract)
 	if err != nil {
 		log.Fatalf("Failed to Deploy or Bind contract: %v", err)
 	}
 
+	err = deployOrBindContract(deployedContract, connBackup, auth, &contractBackup)
```

Comment:
I think here we should not deploy, only bind. Otherwise we deploy two contracts.